gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -719,6 +751,16 @@ gst_tensor_filter_common_free_property (GstTensorFilterPrivate * priv)
 
   gst_tensors_info_free (&priv->in_config.info);
   gst_tensors_info_free (&priv->out_config.info);
+
+  if (priv->stat.recent_latencies != NULL) {
+    GQueue *queue = priv->stat.recent_latencies;
+    gint64 *latency;
+    for (latency = g_queue_pop_tail (queue); latency != NULL;
+        latency = g_queue_pop_tail (queue)) {
+      g_free (latency);
```

Comment:
To make a code more simple, what about using `while` statement as below?
```cpp
while (latency = g_queue_pop_tail (queue)) {
    g_free (latency);
}
```

Reply:
Right! thanks